views/proposals.tmpl

```diff
@@ -118,14 +120,28 @@
                     <tr>
                         <td class="text-left"><a href="/proposal/{{.RefID}}">{{.Name}}</a></td>
                         <td class="text-left">{{.Username}}</td>
-                        {{with .Status}}
-                            <td class="text-left">{{toTitleCase .String}}</td>
-                        {{end}}
                         <td class="text-left">
-                            {{if .VoteStatus}}{{with .VoteStatus}}{{.ShortDesc}}{{end}}{{else}}N/A{{end}}
+                            {{if .VoteStatus}}
+                                {{range $i, $vr := .VoteResults}}
+                                    {{if eq $vr.Option.OptionID "yes"}}
+                                        {{$percent :=  percentage $vr.VotesReceived $v.TotalVotes}}
+                                        {{if lt $percent 60.0}}
```

Comment:
This 60.0 should probably be the `PassPercentage` field in `ProposalInfo`

Reply:
there's an error about non matching data types with using the `PassPercentage`

Reply:
Yeah. It is a little strange that `PassPercentage` is a `uint32` instead of a float. I guess the way you have it is best.

Reply:
You could convert the value obtained from `PassPercentage` and pass into the template in whatever type you want. IMO changing `PassPercentage` value type now might not be worth the effort.

Reply:
Should you decide to pursue this:
> IMO changing PassPercentage value type now might not be worth the effort.

Just change the `PassPercentage` value type all through the code and increase the [semver version](https://github.com/decred/dcrdata/blob/master/gov/politeia/proposals.go#L32) to maybe v1.0.1.
That should fix it just as you need it to be.

Reply:
@dmigwi I actually prefer changing the data type to hard coding the `60.0`, that is assuming there's no external party who may be consuming it as a `uint32`

Reply:
I am not aware of any external party that could be affected as now.

Reply:
Changing the data type of an exported field would bump the major version since it's a breaking change.

I'll ask in the Politeia matrix channel why this is uint32 (https://github.com/decred/politeia/blob/master/politeiawww/api/www/v1/api.md#proposal-vote-status), but only out of curiosity.

Solving this is doable without changing the data type of `gov/politeia/types.ProposalVotes.PassPercentage` or hard coding the value in the template.  I suggest following the example in templates.go of `"int64": toInt64`.  That is, make a `"float64": toFloat64,` entry and then define `toFloat64`.  Except don't make a variable that's a function like was done for `toInt64`; just define a regular function and refer to it by name.